### PR TITLE
dev-libs/libstrophe: drop old versions, backport fix for /bin/sh -> dash issue

### PR DESCRIPTION
**Pull request by NexAdn**
This PR first drops two old versions and backports a fix for build issues when dash is set as `/bin/sh` symlink target.

### CODE REVIEW

File: dev-libs/libstrophe/libstrophe-0.12.2.ebuild [hunk 1]
```diff
@@ -4,7 +4,11 @@
 EAPI=7
 DESCRIPTION="A simple, lightweight C library for writing XMPP clients"
 HOMEPAGE="http://strophe.im/libstrophe/"
-SRC_URI="https://github.com/strophe/${PN}/releases/download/${PV}/${P}.tar.xz"
+# 2nd SRC is a backport of the /bin/sh -> dash fix, #877049, #879533
+SRC_URI="
+	https://github.com/strophe/${PN}/releases/download/${PV}/${P}.tar.xz
+	https://github.com/strophe/libstrophe/commit/7352bd5cdbacf98771fdc0d32a606c4b6718077c.patch
```
Comment (Flowdalic) [inline dev-libs/libstrophe/libstrophe-0.12.2.ebuild:10]:
```suggestion
	https://github.com/strophe/libstrophe/commit/7352bd5cdbacf98771fdc0d32a606c4b6718077c.patch -> libstrophe-fix-configure-bashisms.patch # https://github.com/strophe/libstrophe/pull/218
```

File: dev-libs/libstrophe/libstrophe-0.12.2.ebuild [hunk 2]
```diff
@@ -25,6 +29,10 @@ BDEPEND="
 
 DOCS=( ChangeLog )
 
+PATCHES=(
+	"${DISTDIR}/7352bd5cdbacf98771fdc0d32a606c4b6718077c.patch"
```
Comment (Flowdalic) [inline dev-libs/libstrophe/libstrophe-0.12.2.ebuild:33]:
```suggestion
	"${DISTDIR}/libstrophe-fix-configure-bashisms.patch"  # https://github.com/strophe/libstrophe/pull/218
```

**Review by Flowdalic — CHANGES_REQUESTED** [review]

File: dev-libs/libstrophe/libstrophe-0.12.2.ebuild (hunk 1 above)
Comment (NexAdn) [reply dev-libs/libstrophe/libstrophe-0.12.2.ebuild]: applied

File: dev-libs/libstrophe/libstrophe-0.12.2.ebuild (hunk 2 above)
Comment (NexAdn) [reply dev-libs/libstrophe/libstrophe-0.12.2.ebuild]: applied

**Pull request closed without merging**